src/backend/metal/src/command.rs

```diff
@@ -1689,7 +1695,7 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
     {
         let descriptor = unsafe {
             // FIXME: subpasses
-            let pass_descriptor: metal::RenderPassDescriptor = msg_send![frame_buffer.0, copy];
+            let pass_descriptor = frame_buffer.0.clone();
```

Comment:
Is this change intentional? AFAIK [clone will send `retain` instead of `copy`](https://github.com/gfx-rs/metal-rs/blob/master/src/lib.rs#L388)

Reply:
Sorry, my bad here.
A strong reason to revise the ref-counting model we do for Metal. If we go with the plan expressed in https://github.com/gfx-rs/metal-rs/issues/40 , we can make it so `Clone` is actually cloning the thing (rather than the pointer).